### ISSUE TITLE
Fix iaplayerstat decrement and increment commands, also remove int attribute

### ISSUE THREAD

**Issue by PSYCHEER** [PluginBugs/Issues-ItemsAdder#3839]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

PSYCHEER

### What happened?

Hi! I’m using player stats a lot and I saw a weird thing. TAB completion do not predicate float or int
![image](https://github.com/user-attachments/assets/87748930-6f3a-4291-9fa0-19d9f452cc64)

### Steps to reproduce the issue

Do /iaplayerstat and tab out what it predicates

### Server version

1.20.4 purpur, the very latest

### ItemsAdder Version

3.6.4r3

### ProtocolLib Version

5.2.1 689

### LoneLibs Version

1.0.55 / 1.0.56 doesnt matter in this

### Error (optional)

```shell
No error
```

**Comment by LoneDev6** [comment]
It uses float by default if not specified, is this causing particular issues to your game?
Please elaborate, thanks.

**Comment by PSYCHEER** [comment]
When I don't specify nothing, float or int, the command does nothing. It says value changed but value is not changed

**Comment by LoneDev6** [comment]
Are you using PLAYER_DAT?
```yml
player_stats:
  save_type: PLAYER_DAT
```

**Comment by LoneDev6** [comment]
Anyway I can't reproduce that.
You might have the bypass permission and you might are executing the command from another user to yourself.

https://github.com/user-attachments/assets/d0431287-9474-41c4-b6f6-0c19c87d863c

**Comment by PSYCHEER** [comment]
https://youtu.be/nhGLG8xr41w

It's uploading as GitHub has limitation 10MB and video is 180MB
![image](https://github.com/user-attachments/assets/f55425dc-893f-4cd5-b1c1-93a25cea5a1c)
Issue was created while using v3, here it does too

**Comment by LoneDev6** [comment]
So basically write and read works but not increment, correct?

**Comment by PSYCHEER** [comment]
Increment and/or decrement, yes

**Comment by Tidal-Flame** [comment]
Float doesn't work for the "read" variant of the command. Only int does. Sorry for resurrecting, psycheer redirected me here

An issue I ran into: If your player stat is set to start at 0 and it has not previously existed, the plugin will tell you that the stat was not found, when you try to change its value. I had to circumvent this by giving it a start value of 1. 
My original discord support post was all because of this, not so much because of what the commands would or wouldn't accept for predication. 
![billede](https://github.com/user-attachments/assets/831a0b9d-fa91-447a-ab50-ec124d09c7b6)
![billede](https://github.com/user-attachments/assets/5f92ad83-5b7d-4360-8b57-5a2e62e3f245)
![billede](https://github.com/user-attachments/assets/d2f5ced0-4e64-4908-9f4b-a2ae866c100c)
